vizro-core/examples/_visual-vocabulary/README.md

```diff
@@ -29,83 +29,85 @@ Inspired by:
 Credits and sources:
 
 - Charting library: [Plotly](https://plotly.com/python/plotly-express/)
-
+- Plotly Express chart examples: [Plotly Express](https://plotly.com/python/plotly-express/)
```

Comment:
Why do we link (https://plotly.com/python/plotly-express/) the same thing twice, one below the other?

Reply:
Ups! The first one should be the git repo - I'll change that